lib/boardgamegeek_client.ex

```diff
@@ -5,59 +5,81 @@ defmodule BoardGameGeekClient do
 
   require BoardGameGeek
 
-  def get_game_collection(username, wait \\ 500) do
-    url = "collection?username=#{username}&own=1&subtype=boardgame"
-    doc = get_response(url, wait)
-    game_ids = Exml.get doc, "//items/item/@objectid"
-    Process.sleep(wait)
+  def search_games(query) do
+    doc = "search?query=#{query}&type=boardgame"
+    |> get_response
+
+    Exml.get(doc, "//items/@total")
+    |> String.to_integer
+    |> search_games(doc)
+  end
+  def search_games(0, _), do: []
+  def search_games(_, doc) do
```

Comment:
Are both `search_games/2` functions intentionally public or should these be private since (it seems) like they're only called by `search_games/1`